docs/UserGuide.md

```diff
@@ -714,43 +710,43 @@ Exits the program.
 
-**Q**: Can I edit my data?<br>
+**Q**: Can I edit my data file directly?<br>
 **A**: SociaLite data are saved as a JSON file `<JAR file location>/data/contactlist.json`. Advanced users are welcome to update data directly by editing that data file.
+
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, SociaLite will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, SociaLite will discard all data and start with an empty data file at the next run. You are advised to backup your data before editing the JSON file!
 </div>
 
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command Index <a name="index"></a>
 
-Action | Format | Example (if any)
---------|------------------|--------------------
-**Access Social Media Page** | Click on social media handle | 
-**Add** | `add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]` | `add n/Alex Yeoh p/87438807 t/friends fb/alex.yeoh ig/alex.yeoh tele/alyeoh tiktok/alex.yeoh`
-**Add / Edit Categories** | `edit INDEX [t/TAG]...` | `edit 1 t/Friends`
-**Add / Edit Dates** | `edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​` | `edit 1 date/Meeting:2021-09-14`
-**Add / Edit Profile Picture** | `picture INDEX` | `picture 1`
-**Add / Edit Social Media Handles** | `edit INDEX [p/PLATFORM]` | `edit 1 fb/Alv1n.y`
-**Add Remark** | `remark INDEX r/[TEXT]` | `remark 1 r/Likes to swim`
-**Delete** | `delete INDEX`| `delete 3`
-**Delete All Categories** | `edit INDEX t/` | `edit 1 t/`
-**Delete All Data** | `clear` |
-**Edit** | `edit INDEX [n/NAME] [p/PHONE] [r/REMARK] [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]` | `edit 1 p/91234567 fb/Yalex19`
-**Exit** | `exit` |
-**Find Contacts by Category** | `find t/TAG` | `find t/Friends`
-**Find Contacts by Name** | `find NAME [MORE_KEYWORDS]`| `find James Jake`
-**Find Contacts by Platform** | `find p/PLATFORM` | `find p/facebook`
-**List** | `list` |
-**Pin Contact** | `pin INDEX` | `pin 4`
-**Share Contact** | `share INDEX` | `share 2`
-**Track History** | `↑` or `↓` arrow keys |
-**Unpin Contact** | `unpin INDEX` | `unpin 1`
-**View Help** | `help` |
-**View In-App Guidance** | `help COMMAND` | `help add`
+ Command | Action | Example (if any)
+------------------|--------------------|------------------
+ `add n/NAME p/PHONE_NUMBER ...` | Add a contact | `add n/Alex Yeoh p/87438807 fb/alex.yeoh ig/alex.yeoh` 
+ `clear` | Delete all contacts |  
+ `edit INDEX [k/]...` | Delete optional fields | `edit 1 tiktok/berniceyu` 
+ `edit INDEX [k/KEYWORDS]...` | Edit a contact | `edit 1 t/` 
+ `delete INDEX` | Delete a contact | `delete 3` 
```

Comment:
Just a suggestion as I noticed that you wanted to arrange the Command Index in alphabetical order: Do you want to bring this line forward (position it after `clear`)?